pkg/apis/apm/v1beta1/apmserver_types.go

```diff
@@ -124,12 +124,6 @@ func (as *ApmServer) SecureSettings() []commonv1beta1.SecretSource {
 	return as.Spec.SecureSettings
 }
 
-// Kind can technically be retrieved from metav1.Object, but there is a bug preventing us to retrieve it
-// see https://github.com/kubernetes-sigs/controller-runtime/issues/406
-func (as *ApmServer) Kind() string {
-	return Kind
-}
-
```

Comment:
The `Kind` constant can also be removed in this file.

Reply:
done! forgot due to the v1beta1 merge :stuck_out_tongue_closed_eyes: